Tasktive/Models/ViewModels/TasksViewModel.swift

```diff
@@ -167,6 +167,88 @@ final class TasksViewModel: ObservableObject {
         })
     }
 
+    func deleteTasks(by date: Date, indexSet: IndexSet) async -> Result<Void, UserErrors> {
+        let dateHash = getHashDate(from: date)
+        var mutableTask = tasks
+
+        indexSet.reversed().forEach { taskIndex in
+            let task = tasks[dateHash]!.at(taskIndex)!
```

Comment:
If you're going to force unwrap it anyway, it would read better if you accessed it like this
```swift
let task = tasks[dateHash]![taskIndex]
```